core.js

```diff
@@ -1,8 +1,6 @@
 module.exports = {
 	env: {
-		'shared-node-browser': true,
```

Comment:
completely agree, they should be moved over to the `frameworks` directory- if you feel like doing it in this mister. We could just leave them here for now and just make an issue to move and flesh them out at a later date.